openshift-knative-operator/pkg/serving/secretinformerfiltering.go

```diff
@@ -8,18 +8,13 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"knative.dev/networking/pkg/apis/networking"
+	"knative.dev/networking/pkg/config"
 	"knative.dev/operator/pkg/apis/operator/base"
 	operatorv1beta1 "knative.dev/operator/pkg/apis/operator/v1beta1"
 )
 
-const (
-	// TODO: Remove when available in knative.dev/networking/config
-	ServingInternalCertName = "knative-serving-certs"
-	// TODO: Maybe decide to fetch from net-kourier deps instead
-	EnableSecretInformerFilteringByCertUIDEnv = "ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID"
-	// TODO: Annotation is deprecated, remove in future releases
-	secretInformerFilteringAnnotation = "serverless.openshift.io/enable-secret-informer-filtering"
-)
+// TODO: Maybe decide to fetch from net-kourier deps instead
```

Comment:
I am thinking of moving that const to knative.dev/pkg upstream or something so net-istio could benefit from it. So not eager to bring in the net-kourier dep.

Reply:
Yeah, probably better to move it to pkg upstream. Do you want to create a JIRA for that. We'll probably forget the TODO here.